dist/windows/build_windows_installer.ps1

```diff
@@ -0,0 +1,28 @@
+$name=$args[0]
+$bits=$args[1]
+$meson_options=$args[2]
+$scriptpath=$PSScriptRoot
+$builddir=$(Join-Path -Path $env:TEMP -ChildPath "build-win-installer")
+$installdir=$(Join-Path -Path $env:TEMP -ChildPath "rizin-win-installer")
+
+pushd $PSScriptRoot\..\..
+
+$version=$(python sys\version.py)
+echo $version
+
+$env:Path += ";$env:ProgramFiles (x86)\Microsoft Visual Studio\Installer"
+$env:Path += ";$env:ProgramFiles\7-Zip"
+$env:Path += ";$env:ProgramFiles (x86)\Inno Setup 6"
+
+dist\windows\vsdevenv.ps1 $bits
+
+meson --buildtype=release --prefix=$installdir $builddir $meson_options
+ninja -C $builddir
+ninja -C $builddir install
+7z a dist\windows\Output\rizin-$name-$version.zip $installdir
+iscc dist\windows\rizin.iss /DRizinLocation=$installdir\* /DLicenseLocation=$PWD\COPYING.LESSER /DMyAppVersion=$version
```

Comment:
I think you will need too change also the license path

Reply:
I've already changed it. It was LICENSEbefore and it was a duplicated license inside the rizin-win-installer repository. I'm now using the COPYING.LESSER file that is already in the rizin repo